lib/pbench/common/selinux.py

```diff
@@ -0,0 +1,45 @@
+"""SELinux wrapper module.
+
+There is no "pip" installable `selinux` python module, just a shim, of sorts [1],
+which provides a way to add the installed system's `selinux` module to a
+virtualenv.  But that does not work when the virtualenv uses a version of
+python different from the python version where the real `selinux` module is
+installed.
+
+The only time we need a fully working `selinux` module is when we are not
+running tests.
```

Comment:
I wish that that were obviously true.

In an ideal world, we would test both with SELinux enabled and with it disabled, and we would do that for each of the versions of Python that we support (on each of the platforms that we support).

Assuming that our code will work with SELinux enabled after we've carefully avoided testing that seems a bit of a stretch.  🙁

Reply:
The file tree unit test mocks selinux enabled and disabled; the legacy test system is too fake to be real. We should have a real functional test environment, and, yes one of the variants we should be able to test is selinux on or off ... definitely a point to keep in mind.

Reply:
Removed, not worth having it.